eZ/Bundle/EzPublishCoreBundle/Resources/views/default/full.html.twig

```diff
@@ -0,0 +1,5 @@
+<h2>{{ ez_content_name(content) }}</h2>
+{% for field in content.FieldsByLanguage %}
```

Comment:
nitpick: this "FieldsByLanguage" why starts with upper "F"? Needs to be like that? can't be ```content.fieldsByLanguage```? i